File: tuan02/bai31.jl
using Printf

function thong_ke(a)
    if isempty(a)
        return 0, 0.0000, 0, 0
    end

    tong = sum(a)
    trung_binh = @sprintf("%.4f", Float64(tong) / length(a))
    
    nho_nhat = minimum(a)
    lon_nhat = maximum(a)

    return tong, trung_binh, nho_nhat, lon_nhat
end

File: tuan02/bai31.py
def thong_ke(a):
    if not a:
        return 0, 0.0, 0, 0
    
    tong = sum(a)
    trung_binh = round(tong / len(a), 4)
    nho_nhat = min(a)
    lon_nhat = max(a)
    
    return tong, trung_binh, nho_nhat, lon_nhat
